packing_product_change/README.rst

```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Packing product change
+======================
+
+This module allows you to change product in packing
+
+Usage
+=====
+
+This module adds a button *Replace product* on the stock.move form
+which allows you to change product when packing. The changed product will
+be in invoice but with the price of ogiginal product.
+
+Contributors
+------------
+
+* "Camptocamp, "
+* "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+* "Odoo Community Association (OCA)",
```

Comment:
Here please list the following contributors:

* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com> 
* Guewen Baconnier <guewen.baconnier@camptocamp.com>
* Anar Baghirli